release-notes/DxMs/cloudgateway_change_log.md

```diff
@@ -4,6 +4,12 @@ uid: cloudgateway_change_log
 
 # Cloud Gateway change log
 
+#### 8 August 2024 - Enhancement - CloudGateway 2.14.1 - Syncing improvements [ID_40527]
+
+CloudGateway will no longer sync its cloud endpoint with other DxMs if it does not have internet access. 
+
+In addition, CloudGateway will now also sync the Web API/DMA state with dataminer.services at a regular interval of approximately one minute, even if it had already been synced after a detected change. This will ensure that the connection state remains in sync and is automatically healed, even for Failover systems where sometimes a restart of CloudGateway is needed to get the connection synced and working again.
```

Comment:
```suggestion
In addition, CloudGateway will now also sync the Web API/DMA state with dataminer.services at a regular interval of approximately one minute, even if it had already been synced after a detected change. This will ensure that the connection state remains in sync and is automatically healed, even for Failover systems where before sometimes a restart of CloudGateway was needed to get the connection synced and working again.
```